### PR TITLE
feat KHP3-7225: ETL Operations Management: Implement ETL Operations Dashboard with Log Table and Operations

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
ETL

*None.*

## Related Issue
see: https://thepalladiumgroup.atlassian.net/browse/KHP3-7223
*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-admin-app/src/components/logs-table/operation-log-table.component.tsx [hunk 1]
```diff
@@ -11,88 +11,107 @@ import {
...
-    ...item,
-    id: `row-${index}`,
-  }));
-
   const headers = [
-    { header: t('procedure', 'Procedure'), key: 'procedure' },
-    { header: t('startTime', 'Start time'), key: 'startTime' },
-    { header: t('endTime', 'End time'), key: 'endTime' },
-    { header: t('completionStatus', 'Completion status'), key: 'completionStatus' },
+    { header: t('procedure', 'Procedure'), key: 'script_name' },
```
Comment (donaldkibet) [inline packages/esm-admin-app/src/components/logs-table/operation-log-table.component.tsx:34]:
```suggestion
    { header: t('procedure', 'Procedure'), key: 'scriptName' },
```

Comment (donaldkibet) [reply packages/esm-admin-app/src/components/logs-table/operation-log-table.component.tsx:34]: same for the rest

File: packages/esm-admin-app/src/components/logs-table/operation-log-table.component.tsx [hunk 2]
```diff
@@ -11,88 +11,107 @@ import {
...
+    start_time: formatDate(new Date(item.start_time)),
+    stop_time: formatDate(new Date(item.stop_time)),
+    status: item.status,
+  }));
+
+  const { results: paginatedData, currentPage, goTo } = usePagination(rows, pageSize);
 
   return (
     <div className={styles.table}>
       <CardHeader title={t('etlOperationLog', 'ETL Operations Log')} children={''} />
```
Comment (donaldkibet) [inline packages/esm-admin-app/src/components/logs-table/operation-log-table.component.tsx:52]:
```suggestion
      <CardHeader title={t('etlOperationLog', 'ETL Operations Log')}  />
```

**Review by donaldkibet — APPROVED** [review]

File: packages/esm-admin-app/src/components/logs-table/operation-log-table.component.tsx (hunk 1 above)
Comment (makombe) [reply packages/esm-admin-app/src/components/logs-table/operation-log-table.component.tsx:34]: > same for the rest

@donaldkibet this should come from the endpoint. I think I will reflect the change and once that is done the same can apply for the frontend. Thanks

**Pull request merged**